### PR TITLE
Add “Multiple Xcode Configurations with rules_xcodeproj 1.3” post

### PR DESCRIPTION
**Pull request by brentleyjones**
---

**Version bump**: None.

### CODE REVIEW

File: website/blog/multiple-xcode-configurations-with-rules_xcodeproj-1-3.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+slug: multiple-xcode-configurations-with-rules_xcodeproj-1-3
+title: "Multiple Xcode Configurations with rules_xcodeproj 1.3"
+description:
```
Comment (siggisim) [inline website/blog/multiple-xcode-configurations-with-rules_xcodeproj-1-3.md]: If you want to leave this blank, you can just delete this line

File: website/blog/multiple-xcode-configurations-with-rules_xcodeproj-1-3.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+image: /img/rules_xcodeproj_1_3.png
+tags: [rules_xcodeproj]
+---
+
+Today we released [version 1.3.2][version-1.3] of rules_xcodeproj!
+
+This is a pretty exciting release, as it adds support for multiple Xcode
+configurations (e.g. Debug and Release). Since early in rules_xcodeproj’s
+development, being able to have more than the default Debug configuration has
+been highly requested. We would have implemented support much sooner, bu
```
Comment (tylerwilliams) [inline website/blog/multiple-xcode-configurations-with-rules_xcodeproj-1-3.md:19]: s/bu/but/g

Comment (siggisim) [inline website/blog/multiple-xcode-configurations-with-rules_xcodeproj-1-3.md:20]: bu => but

File: website/blog/multiple-xcode-configurations-with-rules_xcodeproj-1-3.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+a custom transition containing information from
+`xcodeproj.xcode_configurations`. And just like how a solution for a previous
+feature was built upon to enable another feature (i.e. nested invocations which
+enabled isolated build configurations, was built on for generated packages to
+enable multiple Xcode configurations), we should be able to build on this
+solution the same way (e.g. to enable automatic target discovery).
+
+We think the end result is a good starting point, but can be refined futher in
+future releases. Please give it a try, and if you run into any problems
+[file an issue][file-an-issue]! You can also join us in the `#rules_xcodeproj`
```
Comment (tylerwilliams) [inline website/blog/multiple-xcode-configurations-with-rules_xcodeproj-1-3.md:78]: If it's not too complex, wdyt about including a short example snippet demonstrating a minimal Debug / Release configuration somewhere in this post?

**Review by siggisim — APPROVED** [review]
Couple of nits, but overall looks good

**Pull request merged**